Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -550,6 +550,7 @@ - (void)enterFullScreen
     _fullscreenViewController.get().view.frame = _rootViewController.get().view.bounds;
     [self _updateLocationInfo];
 
+#if HAVE(FULLSCREEN_DISMISSAL_GESTURES)
     _startDismissGestureRecognizer = adoptNS([[UISwipeGestureRecognizer alloc] initWithTarget:self action:@selector(_startToDismissFullscreenChanged:)]);
```

Comment:
I don't think guarding this is enough.  There's lots of uses of these ivars in other methods (not to mention the ivars and methods themselves) that we probably want to guard too.

Also, what about `_interactiveDismissTransitionCoordinator`?

Reply:
There were a couple of other places. We didn't really need to guard the ivars or the gesture recognizer calls, but it is cleaner.